robots.js

```diff
@@ -0,0 +1,69 @@
+// Global variables to hold list of all fighters, and user-selected fighter 1 and 2
+let fighters = {};
+let fighter1 = {};
+let fighter2 = {};
+
+// XML call to load fighters
+var data = $.getJSON("characters.json");
+data.then(Main).catch(console.error);
+
+function Main() {
+  // Original Robot prototype
```

Comment:
Not sure what this pattern you're using achieves over the iife syntax. You're defining a wrapper function then invoking it, but you're still exposing the global vars at the top of the file. See if you can reconfigure your JS in a more conventional way that creates a single global object using iifes, so you can have public and private aspects to your code.